cargo-dist-schema/src/snapshots/cargo_dist_schema__emit.snap

```diff
@@ -660,6 +660,8 @@ expression: json_schema
       "type": "object",
       "required": [
         "disable_update_env_var",
+        "github_base_url_env_var",
+        "github_enterprise_base_url_env_var",
```

Comment:
@mistydemeo do we also want to rename that to be `ghe` styled instead of `github_enterprise` or?

Reply:
I was about to say this is purely internal, but it does show up in the manifest where our end users (packagers) will see it. Yeah okay, it should probably be consistent in that case, huh.

(The `GHE` renaming was because `GITHUB_ENTERPRISE` was deemed too long and unwieldy for our grandusers.)